api/defaults.go

```diff
@@ -26,6 +26,20 @@ const defaultKeycardPairingDataFile = "/ethereum/mainnet_rpc/keycard/pairings.js
 
 var paths = []string{pathWalletRoot, pathEIP1581, pathDefaultChat, pathDefaultWallet}
 
+const (
+	statusProdFleet = "status.prod"
+	statusTestFleet = "status.test"
+	wakuv2ProdFleet = "wakuv2.prod"
+	wakuv2TestFleet = "wakuv2.test"
+)
+
+var defaultWakuNodes = map[string][]string{
```

Comment:
Rebase gone wrong? :thinking: Seems unrelated to the changes

Reply:
Sorry, yes, I have added some changes I had locally, will remove